hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -497,6 +679,18 @@ fromLedgerTxWitness wits =
   era =
     ShelleyBasedEraAlonzo
 
+--
+-- Value
+--
+
+liftValue :: Value -> TxOutValue Era
+liftValue =
+  TxOutValue MultiAssetInAlonzoEra
```

Comment:
Should: find a better name. This one is confusing IMO. Analogous to `mkTxOutDatum` this could be named `mkTxOutValue`

Reply:
Which raises the question that maybe we want to have a `mkTxOut :: Plutus.ToData a => ... -> Value -> a -> TxOut CtxTx Era` ?